policyengine_us/variables/gov/states/oh/tax/income/credits/retirement_income/oh_retirement_income_credit.py

```diff
@@ -0,0 +1,33 @@
+from policyengine_us.model_api import *
+
+
+class oh_retirement_income_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Ohio Retirement Income Credit"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://codes.ohio.gov/ohio-revised-code/section-5747.055"
+    defined_for = "oh_retirement_income_credit_eligible"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.oh.tax.income.credits.retirement_income
+
+        person = tax_unit.members
+        is_spouse = person("is_tax_unit_spouse", period)
+        is_head = person("is_tax_unit_head", period)
+        pension_income = person("pension_income", period)
+        has_not_taken_lump_sum_distribution = person(
+            "oh_has_not_taken_oh_lump_sum_credits", period
```

Comment:
do we not model the lump sum credit ($50)?

Reply:
https://casetext.com/statute/ohio-revised-code/title-57-taxation/chapter-5747-income-tax/section-5747055-tax-credit-for-retirement-income (F)

Reply:
We haven't modeled Ohio lump sum credit before, only this placeholder. There is a lump sum retirement credit and a lump sum distribution credit, both one-time credit.

Reply:
> We haven't modeled Ohio lump sum credit before, only this placeholder. There is a lump sum retirement credit and a lump sum distribution credit, both one-time credit.

we can leave the placeholder and file an issue for it. Then we will address that in our next PR